lib/bolt/pal.rb

```diff
@@ -635,6 +635,36 @@ def list_modules
       end
     end
 
+    # Return information about a module.
+    #
+    # @param name [String] The name of the module.
+    # @return [Hash]
+    #
+    def show_module(name)
+      name = name.tr('-', '/')
+
+      data = in_bolt_compiler do |_compiler|
+        mod = Puppet.lookup(:current_environment).module(name.split(%r{[/-]}, 2).last)
+
+        unless mod && (mod.forge_name == name || mod.name == name)
+          raise Bolt::Error.new("Unknown module #{name}", 'bolt/unknown-module')
```

Comment:
```suggestion
          raise Bolt::Error.new("Could not find #{name} on the modulepath. Use #{command} to install the module.", 'bolt/unknown-module')
```
Or something. I'm not sure users will know that this data comes from the modules on the modulepath and not the Forge, so mostly just want to clarify that.

Reply:
Updated. Left out the command suggestion since you can use a short module name (e.g. no author in the name), which doesn't work with the `module add` command.